modules/sdk-coin-avaxp/src/lib/permissionlessValidatorTxBuilder.ts

```diff
@@ -288,8 +288,9 @@ export class PermissionlessValidatorTxBuilder extends TransactionBuilder {
     this.transaction._endTime = permissionlessValidatorTx.subnetValidator.validator.endTime.value();
     this._endTime = this.transaction._endTime;
     this.transaction._fromAddresses = output.outputOwners.addrs.map((a) => a.toBytes());
-    this.transaction._stakeAmount = permissionlessValidatorTx.stake[0].output.amount();
-    this.stakeAmount(permissionlessValidatorTx.stake[0].output.amount());
+    this.transaction._stakeAmount =
+      permissionlessValidatorTx.stake[0].output.amount() - BigInt(this.transaction._network.txFee);
+    this.stakeAmount(this.transaction._stakeAmount);
```

Comment:
the stake amount is sent here https://github.com/BitGo/BitGoJS/blob/eaaa3a803b3968b04d869eee699f3b31bbfce735/modules/sdk-coin-avaxp/src/lib/permissionlessValidatorTxBuilder.ts#L527